### ISSUE TITLE
`Invalid column type Null at index: 1, name: status` on Zome Call

### ISSUE THREAD

**Issue by guillemcordoba** [holochain/holochain#1084]
When making a zome call to a hosted cell, sometimes this error shows up in envoy:

```
Oct 29 10:40:00 hpos holo-envoy[11165]: CONDUCTOR CALL ERROR:
Oct 29 10:40:00 hpos holo-envoy[11165]: {
Oct 29 10:40:00 hpos holo-envoy[11165]:   type: 'error',
Oct 29 10:40:00 hpos holo-envoy[11165]:   data: {
Oct 29 10:40:00 hpos holo-envoy[11165]:     type: 'internal_error',
Oct 29 10:40:00 hpos holo-envoy[11165]:     data: 'Wasm error while working with Ribosome: Host("Invalid column type Null at index: 1, name: status")'
Oct 29 10:40:00 hpos holo-envoy[11165]:   }
Oct 29 10:40:00 hpos holo-envoy[11165]: }
```

I suspect this is the cause that leads to the "Failed handshake" error that the users eventually see on their @holo-host/web-sdk UI.

Weirdly, I don't see this showing up in the holochain logs.

**Comment by freesig** [comment]
Looks like a call that's expecting ops with a validation status is getting some without one.

**Comment by freesig** [comment]
Fixed by #1098
